Assets/Scripts/ROS Scripts/ROS Connections v2/ROSManager.cs

```diff
@@ -61,8 +63,13 @@ public class ROSSensorConnectionInput
         public int port;
         public SensorType sensorType;
         public List<SensorSubscribers> sensorSubscribers;
+        //public List<VideoPlayers> videoPlayers;
+        public VideoPlayers videoPlayers;
     }
 
+    [System.Serializable]
+    public enum VideoPlayers { None, LeftVideo, RightVideo, BackVideo, FrontVideo };
```

Comment:
perhaps this enum belongs in CameraSensor_rosSensorConnection

Reply:
This is is ROSManager so that a user can select which video player they want running at the same place as all the other sensor/subscriber options when they set up.